science/osmocore/Portfile

```diff
@@ -72,6 +71,7 @@ depends_build-append \
 
 depends_lib-append \
     port:gnutls \
+    port:libusb \
```

Comment:
libusb-devel exists, so this should be written using a `path:` dependency.

Reply:
Thanks, see c47a6a239d9. (There appear to be quite a few ports still using `port:libusb`.)